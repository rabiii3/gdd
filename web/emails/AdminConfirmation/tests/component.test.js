import React from 'react';
import { shallow } from 'enzyme';
import { Page } from '..';

describe('web | pages | EmailConfirmation | index', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const props = {
        classes: {},
        tenant: { label: 'label' },
      };
      const wrapper = shallow(<Page {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
