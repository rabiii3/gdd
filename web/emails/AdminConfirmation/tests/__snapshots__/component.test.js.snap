// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | pages | EmailConfirmation | index render should match snapshot 1`] = `
<div>
  <WithStyles(Grid)
    container={true}
  >
    <WithStyles(Grid)
      item={true}
      xs={6}
    >
      <WithStyles(Typography)
        color="textPrimary"
        variant="h6"
      >
        <FormattedMessage
          defaultMessage="Your KOOPT account for {tenant} has been created!"
          id="pages.EmailConfirmation.thanks"
          values={
            Object {
              "tenant": "label",
            }
          }
        />
      </WithStyles(Typography)>
    </WithStyles(Grid)>
  </WithStyles(Grid)>
  <p />
  <WithStyles(Typography)
    align="left"
    color="textPrimary"
    variant="body1"
  >
    <div>
      <FormattedMessage
        defaultMessage="Just click the link below to confirm your account"
        id="pages.EmailConfirmation.getStarted"
        values={Object {}}
      />
       
      <br />
    </div>
    <p />
    <a>
      <FormattedMessage
        defaultMessage="Confirm your account"
        id="pages.EmailConfirmation.getStarted"
        values={Object {}}
      />
    </a>
  </WithStyles(Typography)>
  <p />
</div>
`;
